### PR TITLE
Register starshine.is-a.dev, edit information for previous domain

### PR DESCRIPTION
**Pull request by thaddeuskkr**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] The file's name is all lowercased and alphanumeric. 
- [X] You have completed your website. 
- [X] The website is reachable.  
- [X] You're not using Vercel.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/starshine.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "thaddeuskkr",
+           "email": "",
+           "discord": "275830234262142978",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.lLFfIW3EiPjgK-i-I5E56ERBvjl-ueM91PRyYM6wRClZzuoAj4nQyqL57dSZjhO_4Hbu4ZubbM1I_NI8Vt4_SlsFmxgL-hrvH9tf7rv-hX-TZtBWT_LQQUmhd1ZViGwELTly797M8VXZ39ER2gDmzruxjh8eQEHBCeJujWuUNicigehUgfk-Pk9rSchVlR3TUcwL-wylY58KvBLQwIeMmqUW2qs_E_iKCwk_8FQH-IUb_4W58r6wBHh6MqiP72Py9ssDbxrV59e1eGrQZXYJJM1rUZHoHzW1KX84FgT0QCGc1cSiV-Sdc3p6WceRY6Tf1akfTNjuga61kpgzluoTxg.QwFk8iYMpzefwXgErTr9cQ.kwMwAIrFDxCKfvSxx3Jpvb6ZkiFcp56BFoBg7KQMsYw4PBK60JNzjpMpgMqZ7jymnYJFTz5img3xLlxBF7UuUy_Z9MgHCHaVR3W7970smoQ.aBJoeQFaHiIyXkpQa1Gx7g"
+        },
+    
+        "record": {
+            "A": ["138.2.70.1"]
```
Comment (CuteDog5695) [inline domains/starshine.json:10]: Access denied, please give a preview of your website.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/starshine.json (hunk 1 above)
Comment (thaddeuskkr) [reply domains/starshine.json:10]: oh sorry, i can't have caddy reverse proxy right now cause it tries to obtain an ssl cert and gets ratelimited after failing the challenge.

here's the preview for what both domains are supposed to show:
image

lmk if anything else needed

Comment (CuteDog5695) [reply domains/starshine.json:10]: Ye thats fine, just needed the sites preview, will merge the PR now

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**